test/refactoring/ConvertAsync/basic.swift

```diff
@@ -176,10 +176,18 @@ func alreadyThrows(completion: (String) -> Void) throws { }
 // RUN: not %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1
 func noParamAutoclosure(completion: @autoclosure () -> Void) { }
 
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix BLOCK-CONVENTION %s
+func blockConvention(completion: @convention(block) () -> Void) { }
+// BLOCK-CONVENTION: func blockConvention() async { }
+
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix C-CONVENTION %s
+func cConvention(completion: @convention(block) () -> Void) { }
```

Comment:
Is this meant to be `@convention(c)`?

Reply:
Indeed, that was what I was trying to test.